client/src/store/contentSlice.js

```diff
@@ -173,6 +173,37 @@ export const fetchContentByCountryOrigin = createAsyncThunk(
   }
 );
 
+export const fetchContentByWatch = createAsyncThunk(
```

Comment:
rename addContentByWatch to addContentToWatchHistory

Reply:
Done